hadoop-hdds/pom.xml

```diff
@@ -252,7 +252,15 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <configuration>
           <excludes>
             <exclude>**/hs_err*.log</exclude>
+            <exclude>**/.attach_*</exclude>
+            <exclude>**/**.rej</exclude>
+            <exclude>**/.factorypath</exclude>
+            <exclude>public</exclude>
+            <exclude>**/*.iml</exclude>
             <exclude>**/target/**</exclude>
+            <exclude>output.xml</exclude>
+            <exclude>log.html</exclude>
+            <exclude>report.html</exclude>
```

Comment:
Nit: `hadoop-ozone/pom.xml` has slightly different pattern for these 3 files:

```
            <exclude>**/output.xml</exclude>
            <exclude>**/log.html</exclude>
            <exclude>**/report.html</exclude>
```

Reply:
Yes, that's look better.